index.html

```diff
@@ -230,15 +230,14 @@ <h2><a href="#faq">FAQ</a></h2>
 </dd>
 
 <dt>Q: How does a group prove its documents have had wide review before proceeding to CR?</dt>
-<dd><ul><li>A: See <a rel="nofollow" class="external text" href="https://lists.w3.org/Archives/Public/public-w3process/2014Oct/0133.html">How to work with (experiment with) Wide Review</a> -SNZ</li>
-<li>A: Keep a Disposition of Comments (DoC) Document that shows review comments and acknowledgements that have been received. It is the distribution of the reviewers that shows the review has been wide.-SNZ</li>
-<li>A: Ask the W3C Team if your proposed approach is likely to meet the requirements. -SNZ</li></ul></dd>
+<dd><ul><li>A: Requests to the horizontal review groups will appear in Github, as above. </li>
+<li>A: Keep a Disposition of Comments (DoC) Document showing comments received and a summary of their resolution.</li></ul></dd>
 
 <dt>Q: Is there such a thing as too many reviews?</dt>
 <dd><ul><li>A: Not really</li></ul></dd>
 
 <dt>Q: Is it possible to make too many requests for review?</dt>
-<dd><ul><li> A: unfortunately, yes there is. Multiple requests for reviews can quickly result in a situation of diminishing returns. To help address this, subsequent review requests should clearly identify the exact differences between the last review and current review.</li>
+<dd><ul><li> A: Yes.  To help the review groups, only request updated reviews when substantive changea have been made, and clearly identify the changes since the last review.</li>
```

Comment:
```suggestion
<dd><ul><li> A: Yes.  To help the review groups, only request updated reviews when substantive changes have been made, and clearly identify the changes since the last review.</li>
```